### PR TITLE
Fix downloads page colors

### PR DESCRIPTION
**Pull request by emerick**
Resolves https://github.com/brave/brave-browser/issues/41401

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## UI:

downloads-page
bookmarks-page

### CODE REVIEW

**Comment by emerick** [comment]
@fallaciousreasoning I made this a draft for now, as I figured the way I'm patching this is less than ideal. If you have any thoughts on a better approach, please let me know (would be good if it was a little more generic).

File: ui/webui/resources/br_elements/br_toolbar/br_toolbar.html [hunk 1]
```diff
@@ -103,13 +103,11 @@
   }
 
   .toolbar-extra.-slot-filled {
-    background: rgb(221, 225, 226);
     color: rgb(34, 34, 34);
     padding: 5px;
   }
   @media (prefers-color-scheme: dark) {
     .toolbar-extra.-slot-filled {
-      background: #444;
```
Comment (fallaciousreasoning) [inline ui/webui/resources/br_elements/br_toolbar/br_toolbar.html:112]: Is this change related?

File: patches/chrome-browser-resources-downloads-item.css.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/downloads/item.css b/chrome/browser/resources/downloads/item.css
+index f5c2357c5c986e34e7d1256558036e37ecd2effd..edb2ac5e05bfd1a1be06bdf0c0cff72d2152755e 100644
+--- a/chrome/browser/resources/downloads/item.css
++++ b/chrome/browser/resources/downloads/item.css
+@@ -69,7 +69,7 @@ cr-button {
+ @media (prefers-color-scheme: light) {
+   #content:not(.is-active) {
+     /* GM3 color for "filled cards" */
```
Comment (fallaciousreasoning) [inline patches/chrome-browser-resources-downloads-item.css.patch:8]: oh man, this would be so much easier if they used a CSS variable here :facepalm:

**Review by fallaciousreasoning — COMMENTED** [review]
I'll just see if I can get the Lit overriding stuff working for you again - then it should be a pretty straightforward tweak.

File: ui/webui/resources/br_elements/br_toolbar/br_toolbar.html (hunk 1 above)
Comment (emerick) [reply ui/webui/resources/br_elements/br_toolbar/br_toolbar.html:112]: This is to fix the issues mentioned in these two comments:

https://github.com/brave/brave-browser/issues/41401#issuecomment-2393520226

and 

https://github.com/brave/brave-browser/issues/41401#issuecomment-2397486918

File: patches/chrome-browser-resources-downloads-item.css.patch (hunk 2 above)
Comment (emerick) [reply patches/chrome-browser-resources-downloads-item.css.patch:8]: Yep, I was very aggravated to see the hard-coded rgb value.

**Comment by emerick** [comment]
> I'll just see if I can get the Lit overriding stuff working for you again - then it should be a pretty straightforward tweak.

That would be amazing @fallaciousreasoning!

**Comment by fallaciousreasoning** [comment]
Hopefully should be as simple as adding it to the `injectStyle` style from https://github.com/brave/brave-core/pull/25865 :smile:

File: patches/chrome-browser-resources-downloads-item.css.patch (hunk 2 above)
Comment (fallaciousreasoning) [reply patches/chrome-browser-resources-downloads-item.css.patch:8]: Its mad because they actually define all of these

Comment (fallaciousreasoning) [reply patches/chrome-browser-resources-downloads-item.css.patch:8]: brave://theme/colors.css?sets=ui,chrome

**Review by fallaciousreasoning — APPROVED** [review]
